src/pages/i18n/Drawer.tsx

```diff
@@ -66,14 +76,19 @@ export const Drawer = (props: DrawerProps): JSX.Element => {
             <div className={classes.flexVert} style={{ height: '100%', width: '100%' }}>
                 <div dir={R2L ? 'rtl' : 'ltr'} className={clsx(classes.flexVertBottom, classes.header)}>
                     <BoltIcon style={{ fontSize: '64px', transform: 'rotate(42deg)' }} />
-                    <div style={{ padding: theme.spacing(0.5) }}>
+                    <Typography style={{ padding: theme.spacing(0.5) }}>
```

Comment:
This outer wrapper I would leave as a div

Reply:
Good catch, this shouldn't have been changed.